src/roadmap/common.py

```
@@ -73,20 +73,22 @@ async def query_rbac(
+    url = f"{settings.rbac_url}/api/rbac/v1/access/?{urllib.parse.urlencode(params, doseq=True)}"
 
     try:
-        with urllib.request.urlopen(req) as response:
-            data = json.load(response)
-    except HTTPError as err:
+        async with httpx.AsyncClient(timeout=30) as client:
+            response = await client.get(url, headers=headers)
+            response.raise_for_status()
+            data = response.json()
+    except httpx.HTTPStatusError as err:
         logger.error(f"Problem querying RBAC: {err}")
```

Ruff (0.16.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)